cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/questionnaire/generate/ItemGenerator.java

```diff
@@ -235,16 +234,15 @@ protected IBaseBackboneElement createItemForElement(
 
     protected List<IElementDefinitionAdapter> getElements(
             GenerateRequest request, String parentPath, String sliceName) {
-        List<IElementDefinitionAdapter> elements = new ArrayList<>();
         // Add all elements from the differential
-        elements.addAll(request.getDifferentialElements().stream()
+        List<IElementDefinitionAdapter> elements = new ArrayList<>(request.getDifferentialElements().stream()
```

Comment:
Isn't new ArrayList() redundant?  Why not just to .collect(Collectors.toList()), which will return a mutable List, if that's what you want.